### PR TITLE
Update dependencies to use PHP7.0 and contentful-php 4.0 sdk.

### PR DESCRIPTION
**Pull request by yadakhov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ git clone https://github.com/contentful/boilerplate-php.git
 
 Open your terminal, navigate to this folder and run `./run.sh`. That's it!
 
+### To run from CLI
```
Comment (mariobodemann) [inline README.md]: Hey,

I am not 100% sure, but I see the same calls in the `./run.sh`. Should we maybe only have one recommendation on how to setup the boilerplate from a cli?

File: index.php [hunk 2]
```diff
@@ -7,12 +7,15 @@
 $spaceID = 'cfexampleapi';
 $accessToken = 'b4c0n73n7fu1';
 
+// Feel free to customize this snippet as much as you like.
```
Comment (mariobodemann) [inline index.php]: :heart_eyes:

Comment (mariobodemann) [inline index.php]: :+1:

File: README.md (hunk 1 above)
Comment (yadakhov) [reply README.md]: Ok.  I will remove this comment as it may be confusing with the contentful cli.

**Pull request merged**